ui/app/models/structured-attributes.js

```diff
@@ -8,6 +8,11 @@ const { unflatten } = flat;
 export default class StructuredAttributes extends Fragment {
   @attr() raw;
 
+  recomputeRawProperties(incoming) {
+    this.set('raw', incoming);
+    this.notifyPropertyChange('structured');
```

Comment:
`this.set` [internally calls](https://discuss.emberjs.com/t/notifypropertychange-behavior/14981) `this.notifyPropertyChange`.

I think the issue is that we're not setting `raw` to a new reference in memory since we're just setting it to `incoming`.

I think we might be able to get away with:

```
  recomputeRawProperties(incoming) {
    this.set('raw', {...incoming}); // this will set raw to a new reference in memory that should trigger the dependency key to notice that a change actually occured
  }

```

Reply:
Good call, that's cleaner.